packages/web-components/fast-foundation/src/anchored-region/intersection-service.ts

```diff
@@ -50,6 +55,10 @@ export class IntersectionService {
      * initialize intersection detector
      */
     private initializeIntersectionDetector = (): void => {
+        if (!window.IntersectionObserver) {
```

Comment:
Instead of using `window` import `$global` from `fast-element`. It's a platform safe global you can use for these sorts of checks.

Reply:
Cool, done.